packages/code-studio/src/main/AppInit.jsx

```diff
@@ -96,14 +97,16 @@ const AppInit = props => {
     log.debug('Loading plugins...');
     try {
       const manifest = await PluginUtils.loadJson(
-        `${process.env.REACT_APP_MODULE_PLUGINS_URL}/manifest.json`
+        `${import.meta.env.VITE_MODULE_PLUGINS_URL}/manifest.json`
```

Comment:
Have you tested this with plugins? E.g. the matplotlib plugin?

Reply:
It "works". The issue is not on the code-studio side, but on the plugin side.

The plugins need to not externalize reactstrap because it's been removed from code-studio.